openassessment/templates/openassessmentblock/grade/oa_grade_waiting.html

```diff
@@ -16,13 +16,7 @@ <h2 class="step__title">
         <div class="wrapper--step__content">
             <div class="step__content">
                 <div class="grade__value__description">
-                    {% if waiting == 'peer' %}
-                        <p>{% trans "Your response is still undergoing peer assessment. After your peers have assessed your response, you'll see their comments and receive your final grade." %}</p>
-                    {% elif waiting == 'example-based' %}
-                        <p>{% trans "Your response is still undergoing example-based assessment. After your response has been assessed, you'll see the comments and receive your final grade." %}</p>
-                    {% elif waiting == 'all' %}
-                        <p>{% trans "Your response is still undergoing peer assessment and example-based assessment. After your peers have assessed your response and example-based assessment is complete, you'll see your peers' comments and receive your final grade." %}</p>
-                    {% endif %}
+                    <p>{% trans "Your response still needs to undergo some assessment steps. After all assessment steps are complete, you will see comments from peers or course staff and receive your final grade." %}</p>
```

Comment:
@explorerleslie This is the suggestion that @catong made in https://github.com/edx/edx-ora2/pull/764. Here is a screenshot, the user has finished training, self assessment, and their peer assessment requirements, but no peers or staff have graded the user.

<img width="871" alt="screen shot 2016-01-04 at 4 24 40 pm" src="https://cloud.githubusercontent.com/assets/7373924/12100947/3931b626-b300-11e5-8c6c-3694d1067f37.png">

Because we're trying to avoid a crazy amount of strings to enumerate, the display does not change at all after peer grades for the user are made, since we're still waiting on staff assessment.